From 02777f6a4a896fe5672e23411bc20df597266a2a Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sat, 22 Dec 2018 18:58:43 +0000
Subject: [PATCH 4/7] klte-common: Enable memory optimizations

 * Delay service restart for devices with 2GB or less memory.

Change-Id: Iee114d6c83a09d90ce68c6bcb971596c4df6cc74
---
 system.prop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/system.prop b/system.prop
index d85db0b..67cc46c 100644
--- a/system.prop
+++ b/system.prop
@@ -41,6 +41,9 @@ ro.gps.agps_provider=1
 ro.qc.sdk.izat.premium_enabled=0
 ro.qc.sdk.izat.service_mask=0x0
 
+# Memory optimizations
+ro.vendor.qti.am.reschedule_service=true
+
 # NFC
 ro.nfc.port=I2C
 
-- 
2.17.1

